<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>custom_model_widget</class>
 <widget class="QWidget" name="custom_model_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QScrollArea" name="custom_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>228</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="scan_for_models_button">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scan for models</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="toggle_all">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Toggle all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>scan_for_models_button</sender>
   <signal>clicked()</signal>
   <receiver>custom_model_widget</receiver>
   <slot>action_button_clicked_scan_for_models()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_button_clicked_scan_for_models()</slot>
 </slots>
</ui>
